### DISCUSSION TITLE
Is it possible to create users using the API

### DISCUSSION THREAD

**Discussion by socekin** [FreshRSS/FreshRSS#6647] [Q&A] [answered]
I have such a requirement, in my application, allow users to add RSS, and then help users aggregate RSS Feeds. I want to use FreshRSS to set up such a service, is there an API to realize the creation of users in FreshRSS and get the URL corresponding to the user's RSS aggregation? Thank you.

**Comment by Alkarex** [comment, marked as answer]
There is a command line allowing the creation of users https://github.com/FreshRSS/FreshRSS/blob/edge/cli/README.md
